doc/smart-switch/pmon/smartswitch-pmon.md

```diff
@@ -0,0 +1,947 @@
+swss:portsyncd            OK        Process
+swss:coppmgrd             OK        Process
+swss:tunnelmgrd           OK        Process
+bgp:zebra                 OK        Process
+bgp:staticd               OK        Process
+dpu-pdsagent              OK        UserDefine
+dpu-pciemgrd              OK        UserDefine
+dpu-eth_Uplink1/1_status  OK        UserDefine
+dpu-pcie_link             OK        UserDefine
+
+System services and devices ignore list
+
+Name    Status    Type
+------  --------  ------
+fan     Ignored   Device
+psu     Ignored   Device
+```
+#### Vendor specific health checkers
+* If the vendor wants to add additional information specific to the platform it can be done using the user-defined checkers:
+* The DPU specific interrupt events can be part of this.
+
+show system-health monitor-list       <font>**`Executed on the switch`**</font>
+```
+System services and devices monitor list
+
+Name                       Status    Type
+-------------------------  --------  ----------
+…
+bgp:bgpcfgd                OK        Process
+CPU                        OK        UserDefine
+DDR                        OK        UserDefine
+```
+
+show interface status - will show the NPU-DPU interface status also      <font>**`Executed on the switch`**</font>
+```
+root@sonic:~# show interfaces status
+  Interface                                    Lanes    Speed    MTU    FEC    Alias    Vlan    Oper    Admin    Type    Asym PFC
+-----------  ---------------------------------------  -------  -----  -----  -------  ------  ------  -------  ------  ----------
+  Ethernet0  2816,2817,2818,2819,2820,2821,2822,2823     400G   9100    N/A     etp0  routed    down       up     N/A         N/A
+  Ethernet8  2824,2825,2826,2827,2828,2829,2830,2831     400G   9100    N/A     etp1  routed    down       up     N/A         N/A
+ Ethernet16  2056,2057,2058,2059,2060,2061,2062,2063     400G   9100    N/A     etp2  routed    down       up     N/A         N/A
+ Ethernet24  2048,2049,2050,2051,2052,2053,2054,2055     400G   9100    N/A     etp3  routed    down       up     N/A         N/A
+ Ethernet32  1792,1793,1794,1795,1796,1797,1798,1799     400G   9100    N/A     etp4  routed    down       up     N/A         N/A
+ Ethernet40  1800,1801,1802,1803,1804,1805,1806,1807     400G   9100    N/A     etp5  routed    down       up     N/A         N/A
+ ...
+ ...
+
+### These are internal DPU ports ###
+...
+...
+Ethernet240                                  4,5,6,7     100G   9100    N/A   etp30a  routed    down       up     N/A         N/A
+Ethernet244                                  0,1,2,3     100G   9100    N/A   etp30b  routed    down       up     N/A         N/A
+Ethernet248                                8,9,10,11     100G   9100    N/A   etp31a  routed    down       up     N/A         N/A
+Ethernet252                              12,13,14,15     100G   9100    N/A   etp31b  routed    down       up     N/A         N/A
```

Comment:
will the internal DPU ports name be aligned with the show command output which executed from DPU? I am asking because I see the name is different with the below command executed from DPU.

Reply:
Fixed it.  Thanks for mentioning this!